sql/moz-fx-data-shared-prod/search_terms/sanitization_job_data_validation_metrics/view.sql

```diff
@@ -0,0 +1,26 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.search_terms.sanitization_job_data_validation_metrics`
+AS
+SELECT
+        finished_at,
+        total_search_terms_removed_by_sanitization_job / total_search_terms_analyzed AS pct_sanitized_search_terms,
+        contained_at / total_search_terms_analyzed AS pct_sanitized_contained_at,
+        contained_numbers / total_search_terms_analyzed AS pct_sanitized_contained_numbers,
+        contained_name / total_search_terms_analyzed AS pct_sanitized_contained_name,
+        sum_terms_containing_us_census_surname / total_search_terms_analyzed AS pct_terms_containing_us_census_surname,
+        sum_uppercase_chars_all_search_terms / sum_chars_all_search_terms AS pct_uppercase_chars_all_search_terms,
+        sum_words_all_search_terms / total_search_terms_analyzed AS avg_words_all_search_terms,
+        1 - languages.english_count / languages.all_languages_count AS pct_terms_non_english
```

Comment:
If any of the divisors can be 0 then I would recommend using `SAFE_DIVE` (to cast to null) otherwise querying the view will fail

Reply:
Good call. Thank you! CHange applied.